rds-dc-with-one-vm-domainjoin/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Domain join sample
+
+This template demonstrates domain join to a private AD domain up in cloud. 
+It creates one DC VM, one other VM and joins it to the domain.
+
+This template deploys the following resources:
+<ul><li>storage account</li><li>vnet</li><li>public ip</li><li>load balancer</li><li>a DC virtual machine</li><li>and another virtual machine</li></ul>
+
+
+Click the button below to deploy
+
+<a href="https://azuredeploy.net" target="_blank">
```

Comment:
We've deprecated this azuredeploy code. Please take a look at another template on the new way to deploy to azure straight to Ibiza portal